tech/languages/ruby/gems-installation.md

```diff
@@ -73,3 +73,32 @@ To look whether the required gem is available as an RPM package, look at
 [IsItFedoraRuby.com](http://isitfedoraruby.com/) site which tracks the Ruby
 integration within the Fedora project by listing available information about
 packaged gems.
+
+## Configuring RubyGems
+
+Fedora configures ruby to install gems on your user's directory; as opposed to the system's, by default. The gems get installed at: `$HOME/.local/share/gem`. 
+
+One can check the defaults [here](https://src.fedoraproject.org/rpms/ruby/blob/rawhide/f/operating_system.rb).
+
+If one would like, say, to disable documentation generation at install time, one could do it in two ways:
+
+```
+install: --no-document
+```
+
+This method won't override the system's defaults. 
+
+Or, in order to override the system's default, one can use:
+
+```
+gem: --user-install
+```
+Remember to add: `$HOME/.local/share/gem/ruby/bin` to your `$PATH` if you do since all settings aforementioned will be overriden. 
+
+One way of doing this is adding the following to `$HOME/.bash_profile`: 
+
+```
+# my path
+## ruby
+export PATH="$HOME/.local/share/gem/ruby/bin:$PATH"
+```
```

Comment:
Please also explain to the user what is this doing.

I.e. 'This will add the directory to the list of paths that shell is searching for binaries."

IOW something that is closer to this style: https://developer-portal.github.io/contributing/write-general-writing-hints.html#tell_the_user_whats_going_on

Reply:
I thought I did; in line 96.